components/form-builder/app/edit/elements/DropDown.tsx

```diff
@@ -43,7 +43,9 @@ export const DropDown = ({
         {isOpen &&
           items.map((item, index) => (
             <li
-              className={highlightedIndex === index ? "highlighted" : ""}
+              className={
+                highlightedIndex === index ? `highlighted ${item.className}` : item.className
```

Comment:
This used to append a separator <div> that broke in a previous styling update.

 Now just passing a className in so we can style without the extra div